backend/Cargo.toml

```diff
@@ -13,6 +13,7 @@ console_error_panic_hook = "0.1.6"
 console_log = "1.0.0"
 contour = "0.12.0"
 fast_paths = "1.0.0"
+flate2 = "1.1.1"
```

Comment:
Temporary hack, this should be dev-dependencies; it's only for the benchmark code. I think we need to follow the note in `lib.rs`:

```
// TODO: We could hide this behind a feature flag - it's used by both tests and benches
pub mod test_fixtures;
```

I couldn't figure out how to feature-guard for "in test/benchmark mode". I'll wire this up in a separate followup